Modules/Database/Database.lua

```diff
@@ -5,11 +5,34 @@ local LOG = CLM.LOG
 
 local DB = {}
 
+-- You really do not want to modify this
+-- vvvvv
+local DB_NAME_PERSONAL = 'personal'
+local DB_NAME_GUI = 'gui'
+local DB_NAME_RAID = 'raid'
+local DB_NAME_GUILD = 'guild'
+local DB_NAME_LEDGER = 'ledger'
+local DB_NAME_LOGGER = 'logger'
+local DB_NAME_GLOBAL = 'global'
+-- ^^^^^
+
 local function UpdateGuild()
     DB.server_faction_guild = string.lower(UnitFactionGroup("player") .. " " .. GetNormalizedRealmName() .. " " .. (GetGuildInfo("player") or "unguilded"))
     LOG:Debug("Using database: %s", DB.server_faction_guild)
 end
 
+local function UpdateSchema(table, schema)
+    if type(schema) == "table" then
+        for key, value in pairs(schema) do
+            if not table[key] then
+                table[key] = value
```

Comment:
This is risky, when a value is of type table it will be copied by reference not by value. This means updates will update the schema.

This should probably be a deep copy.

Reply:
changed to deep copy 👍